synapse/storage/databases/main/relations.py

```diff
@@ -515,6 +545,98 @@ def _get_if_user_has_annotated_event(txn: LoggingTransaction) -> bool:
             "get_if_user_has_annotated_event", _get_if_user_has_annotated_event
         )
 
+    async def _get_bundled_aggregation_for_event(
```

Comment:
Is this brand new logic? Ah, no, it's moved from `_injected_bundled_aggregations` in the next commit.

Reply:
Yes, it is mostly copied! Sorry about not making that clear!